Package.swift

```diff
@@ -2,6 +2,13 @@
 
 import PackageDescription
 
+let macOSOnlyDependency: [Package.Dependency]
+#if !os(Linux)
```

Comment:
Any reason not to do a mac check here instead to be more explicit?

Reply:
In CI machine, we cannot build my tools, I'm checking the reason

Reply:
fixed